### ISSUE TITLE
NWPTA (nuans) changes

### ISSUE THREAD

**Issue by katiemcgoff** [bcgov/name-examination#653]
~~Have meeting to review, then implement inevitable changes. TBD.~~
~~Need clarity around xpro, jurisdiction, nwpta. Specifically what needs to be supported at release time for x ammt of time (6 months? year?)~~

Finish NWPTA display and edit.

**Comment by katiemcgoff** [comment]
NWPTA stuff to do:
- [ ] additional info canned text
- [ ] type code - AS/A/???
- [ ] check saving it in Oracle files (waiting on Thor?)

**Comment by katiemcgoff** [comment]
## Assumptions made during implementation: ##

- examiner cannot change which type of record this is, ie: from number (CO or LP), assumed name (AS), and assumed number (NAS).
- examiner cannot create a new nwpta record where one does not exist, ie: can't add a Saskatchewan record if it wasn't either requested (as a do-it-for-me request) or filled in by customer.
- we do not want to remove the data in the case of switching from a request type that uses/displays nwpta and one that does not. Ie: we don't clear the data if we switch away from a request type that does not need it. We just ignore it.
- all Additional Info canned text will be filled in by NRO when submitted by customer, and Namex will not adjust that automatically - ie: we have no need to add to it because we're not creating nwpta records that don't already exist. And we won't delete it from Additional Info, the examiner can do that manually if it is no longer relevant.
